### ISSUE TITLE
Trailing comment results in unmached braces

### ISSUE THREAD

**Issue by jmiskovic** [kkinnear/zprint#106]
Using **zprintl-0.4.15** on following code results in incorrect formatting:
```
(defn foo
  [bar]
  (case bar
    :a 1
    :b 2
    3 ;default case
    ))
```
The result omits two closing braces:
```
(defn foo
  [bar]
  (case bar
    :a 1
    :b 2
    3 ;default case
```
The obvious workaround is to place the comment on its own line, which produces correct code.

**Comment by kkinnear** [comment]
That's terrible!  Thank you so much for finding and reporting this.  People clearly don't comment the default clause in the case much.   I have a fix for this, and I will try to release it in the next couple of weeks.  It is becoming clear that I can't wait for a release until I have finished the major new feature on which I am working, so I will work on gathering up the fixes that I have in progress and work to get them out.  Thank you again for reporting this!

**Comment by kkinnear** [comment]
Fixed in version 0.4.16.

**Comment by jmiskovic** [comment]
Works great, thanks.
